### ISSUE TITLE
[ENHANCEMENT] Distinct font colors for link-based environments

### ISSUE THREAD

**Issue by PhilHem** [Zettlr/Zettlr#923]
## Description

I've been starting to think a little bit about how I can improve on the general readability of Zettlr. 
Certainly, this is very much a personal thing, but I'd like to adress the basic style philosophies that underly the presets. 
One thing that I always found a little difficult is that for example links and Wikilinks have the same color as emphasis environments. 
I find they serve a different purpose and therefore should be more visually distinct.

Here is an example of what it could look like: 

![image](https://user-images.githubusercontent.com/34508934/84564797-d7f95d00-ad64-11ea-8393-7b30b9d0ce45.png)

I also subdued the colors a little bit. I think they should just be visually distinct enough to be helpful for seeing emphasis, and nothing more than that.

## Caveats

I admit, there might be use cases where people might prefer the same looks for emphasis and links. I just think, it should not be the default case. 

## Additional Information
This is highly personal topic and I am certainly no style authority, so I'm up for a healthy discussion :)

**Comment by nathanlesage** [comment]
Isn't this issue more of a discussion? If so, maybe the tag `[DISCUSSION]` in the title might help!

**Comment by PhilHem** [comment]
Haha, I wasn't sure if a [DISCUSSION] was a thing here on GitHub :) But sure

**Comment by nathanlesage** [comment]
Well … it's highly encouraged to do so on the forum, but I gave up directing all people like a traffic policing officer 😄

**Comment by PhilHem** [comment]
I'd very much wish for something like a poll on matters of taste. It feels impossible to actually get a significant mass of people to chime in on something like this...

**Comment by nathanlesage** [comment]
> It feels impossible to actually get a significant mass of people to chime in on something like this...

Yes, that's true … what would be needed would be a user survey that just collects all these things and asks all users in bulk about them. But therefore we'd need someone to collect all these possible matter-of-taste-things and then create a survey from it. I mean, the Zettlr server already has a survey tool, but the survey is missing

**Comment by PhilHem** [comment]
Would it be an option to ask the user of Zettlr when updating to a new version, to subscribe to polls that will be conducted via the app, if the user confirms that he wants to contribute to the development of Zettlr? I think single quick question would be concieved better than long polls that take more then just a quick click. And then, maybe it would make sense to limit the polls to  once a week or so. 
I think this could be a great way to have more people contribute very easily.
I'm thinking of it like the polls Youtube shows for advertisements from time to time. (I never answer those, but still... :P )

**Comment by nathanlesage** [comment]
Ehm … do you want to code all that stuff both within Zettlr and within the API running on the server? o_o Then please, but I am *extremely* sure that this is not feasible from a cost/effectiveness-perspective …

**Comment by PhilHem** [comment]
Oh my understanding of the work involved to realize something like this is faint :P 
Maybe one simpler way to do it would be an E-Mail List to subscribe to!

**Comment by nathanlesage** [comment]
> Maybe one simpler way to do it would be an E-Mail List to subscribe to!

Hmhm … or just … use Twitter? I don't know but I have the feeling the most part of the user base is there already, so we'd grab the most and I've had pretty good experiences with polls over there in the past.

Maybe it's really finally time to activate the Discord-server I've kept set up …

**Comment by PhilHem** [comment]
Could be adequate! 
Personally, I try to stay away from social media platforms like Twitter, but I could imagine that a large numer of Zettlr users use it. (Can I call them Zettlrs?)

Today though, as the new version of the beta was distributed, I did simply get a notification within Zettlr. Couldn't a notification like that be used to link to the appropriate poll?

**Comment by nathanlesage** [comment]
> Couldn't a notification like that be used to link to the appropriate poll?

On new releases, surely! But implementing it independent of releases would be very hard
